connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookConnectorExecutable.java

```diff
@@ -0,0 +1,27 @@
+package io.camunda.connector.runtime.inbound.webhook;
+
+import io.camunda.connector.api.annotation.InboundConnector;
+import io.camunda.connector.api.inbound.InboundConnectorContext;
+import io.camunda.connector.api.inbound.InboundConnectorExecutable;
+import io.camunda.connector.impl.inbound.InboundConnectorProperties;
+
+@InboundConnector(name = "WEBHOOK", type = WebhookConnectorRegistry.TYPE_WEBHOOK)
+public class WebhookConnectorExecutable implements InboundConnectorExecutable {
```

Comment:
I am wondering, whether we somehow could move (not in this PR but in another one) this to `connectors-bundle` repository, since all `@OutboundConnector`-s are there.

Reply:
Currently it's not possible, because `WebhookConnectorExecutable` depends on runtime-specific components [`WebhookConnectorRegistry`](https://github.com/camunda-community-hub/spring-zeebe/blob/5972ea8833eb160fa561829277f0b818556e927c/connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/webhook/WebhookConnectorRegistry.java), and transitively [`InboundWebhookRestController`](https://github.com/camunda-community-hub/spring-zeebe/blob/5972ea8833eb160fa561829277f0b818556e927c/connector-runtime/src/main/java/io/camunda/connector/runtime/inbound/webhook/InboundWebhookRestController.java), which can only exist in Runtime.

I think it may be possible later on, but not sure if it makes sense - since the actual endpoint code will always live in the runtime, the executable can't be used by itself, separately from the runtime.